lab/SoundTest/example_scores.js

```diff
@@ -91,25 +90,25 @@ export function layered_melody() {
     ])
   );
 
-  const sine_part = new MusicLoop(new Rational(34, 1), pedal);
+  const sine_part = Melody.from_repeat(pedal, 34);
```

Comment:
Not 34 times! `34 measures / pedal.duration` times. I think we're only getting half the length.